### DISCUSSION TITLE
How to view the number of online editor connections?

### DISCUSSION THREAD

**Discussion by SizerSong** [ONLYOFFICE/DocumentServer#1798] [Q&A] [answered]
Hi,
I want to Buy Enterprise Edition, but i don't know how many connections i need, Is there any way to monitor the onlyoffice editor connections?

**Comment by ShockwaveNN** [comment, marked as answer]
There is statistics accessible only from localhost of your server at
```
curl localhost/info/info.json
```

Also you can modify `/etc/onlyoffice/documentserver/nginx/includes/ds-docservice.conf` 
part 
```
# Allow "/info" interface only from 127.0.0.1 by default
# Comment out lines allow 127.0.0.1; and deny all; 
# of below section to turn on the info page
location ~* ^(\/[\d]+\.[\d]+\.[\d]+[\.|-][\d]+)?\/(info)(\/.*)$ {
  allow 127.0.0.1;
  deny all;
  proxy_pass http://docservice/$2$3;
}
```

To be accessible from outside and it will be shown at `http://docserver_url/info`

Or you can contact our sales team and they help you decide that numbers are best suited for you

**Reply by SizerSong** [reply]
thanks for your reply, i'm going to try it now

**Reply by garciadavy** [reply]
Are they simultaneous connections in the "connectionStats" part ?
What is the "connectionsCount" quota ?
Thank you for your help.
